src/Processors/Content/ContentCore.php

```diff
@@ -460,7 +461,7 @@ public static function parseTitle( string $title ) {
 						$title
 					);
 				} elseif ( $fn !== '' ) {
-					if ( Config::getConfigVariable( 'create-seo-titles' ) === true ) {
+					if ( Config::getConfigVariable( 'create-seo-titles' ) === true && $noSEO === false ) {
```

Comment:
This can be `&& !$noSEO`, since the `bool` type hint guarantees it is a boolean.